spec/support/vcr.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require "vcr"
+
+ENV["ATRIUM_API_KEY"]   = ENV.fetch("ATRIUM_API_KEY",   "ATRIUM_API_KEY")
+ENV["ATRIUM_CLIENT_ID"] = ENV.fetch("ATRIUM_CLIENT_ID", "ATRIUM_CLIENT_ID")
+
+VCR.configure do |config|
+  config.cassette_library_dir = "spec/cassettes"
+  config.hook_into :webmock
+
+  config.configure_rspec_metadata!
+
+  config.filter_sensitive_data("ATRIUM_API_KEY")   { ENV["ATRIUM_API_KEY"] }
+  config.filter_sensitive_data("ATRIUM_CLIENT_ID") { ENV["ATRIUM_CLIENT_ID"] }
+end
+
+RSpec.configure do |config|
+  # Add VCR to all tests
+  config.around(:each) do |example|
+    options = example.metadata[:vcr] || {}
+
+    if options[:record] == :skip
+      VCR.turned_off(&example)
+    else
+      name = example
```

Comment:
By default, VCR requires manual management of the cassettes. This becomes a huge stateful pain. This bit of magic takes all the `describe`, `context`, and `it` strings, binds them together into a file path/tree. Allowing the automatic creation and management of cassettes in any and all situations. The vast majority of this is to make the folder and file names valid.